File: 3774-absolute-difference-between-maximum-and-minimum-k-elements/3774-absolute-difference-between-maximum-and-minimum-k-elements.cpp
class Solution {
public:
    int absDifference(vector<int>& nums, int k) {
        if(k>nums.size()) return 0;
        sort(nums.begin(),nums.end());
        int ans=0;
        for(int i=0;i<k;i++){
            ans+=nums[nums.size()-i-1]-nums[i];
        }
        return abs(ans);
    }
};

File: 3774-absolute-difference-between-maximum-and-minimum-k-elements/3774-absolute-difference-between-maximum-and-minimum-k-elements.java
class Solution {
    public int absDifference(int[] nums, int k) {
        if(k>nums.length) return 0;
        Arrays.sort(nums);
        int ans=0;
        for(int i=0;i<k;i++){
            ans+=nums[nums.length-i-1]-nums[i];
        }
        return Math.abs(ans);
    }
}